test-example-diagrams/src/main/resources/merging-inclusive/MergingInclusive.md

```diff
@@ -0,0 +1,28 @@
+# Merging Inclusive
```

Comment:
Looking at the diagram, the title should be **Merging Parallel**?

Reply:
No, this is an Inclusive gateway here in the diagram ( ? )
![image](https://user-images.githubusercontent.com/5063808/224113710-2a50a3a0-5931-461b-aaad-7f9330efe72f.png)

Reply:
Sorry, I just looked at the png, and there is a parallel gateway on merging:

![image](https://user-images.githubusercontent.com/81753891/224258946-6e03e8d3-d9d8-4476-9731-fa9f3149ecde.png)